modules/auth/auth.service.js

```diff
@@ -53,27 +60,42 @@ const register = async (body) => {
   const existedUsername = await userService.findByUsername(body.username);
 
   if (existedUsername) {
-    throw new Error('User has been used!');
+    throw new APIError({
+      message: 'User has been used!',
+      status: 409
+    });
   }
 
   const existedEmail = await userService.findByEmail(body.email);
 
   if (existedEmail) {
-    throw new Error('Email has been used!');
+    throw new APIError({
+      message: 'Email has been used!',
+      status: 409
+    });
   }
 
   const existedPhonenumber = await userService.findByPhonenumber(body.phoneNumber);
 
   if (existedPhonenumber) {
-    throw new Error('Phone number has been used!');
+    throw new APIError({
+      message: 'Phone number has been used!',
+      status: 409
+    });
   }
 
   if (body.age < COMMON.ALLOWED_AGE) {
-    throw new Error('You must be 18+ to play this game!')
+    throw new APIError({
+      message: 'You must be 18+ to play this game!',
+      status: 400
```

Comment:
I think it should be `403`
`403 `Forbidden:
The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.

The player below 18 does not have access rights to play game